api/src/main/java/org/openmrs/module/UpdateFileParser.java

```diff
@@ -103,7 +103,7 @@ public void parse() throws ModuleException {
 				this.currentVersion = "";
 				
 				// loop over all 'update' tags
-				for (Integer i = 0; i < nodes.getLength(); i++) {
+				for (int i = 0; i < nodes.getLength(); i++) {
```

Comment:
Thanks @sacull for this , why didnt you first make an integer a variable first, something like For (Integer int )

Reply:
@sherrif10  Integer and int are all data types,they cant be used together.Integer is for object and int is for primitive.